client/landing/stepper/declarative-flow/design-first.ts

```diff
@@ -55,7 +55,7 @@ const designFirst: Flow = {
 			},
 			{
 				slug: 'domains',
-				asyncComponent: () => import( './internals/steps-repository/choose-a-domain' ),
```

Comment:
Yes, thank you for checking. I have noted that in the PR description. I am making this change carefully to avoid issues in production and mainly because I could figure why we are using the `choose-a-domain` component.